### ISSUE TITLE
is this project still open?

### ISSUE THREAD

**Issue by danny95928** [pemami4911/neural-combinatorial-rl-pytorch#22]
can't find the SGD in criticnetwork and cant find the temperture in softmax
